components/console-backend-service/internal/domain/rafter/assetgroup_resolver.go

```diff
@@ -28,7 +28,7 @@ func newAssetGroupResolver(assetGroupService assetGroupSvc, assetGroupConverter
 	}
 }
 
-func (r *assetGroupResolver) AssetGroupAssetsField(ctx context.Context, obj *gqlschema.AssetGroup, types []string) ([]gqlschema.RafterAsset, error) {
+func (r *assetGroupResolver) AssetGroupAssetsField(ctx context.Context, obj *gqlschema.AssetGroup, types []string) ([]gqlschema.Asset, error) {
```

Comment:
So, this resolver operates on `Asset`s. Why the domain is called `rafter` then? Wouldn't `assets` be more clear and meaningful?

Reply:
It's called `rafter` because in this domain we operate on `rafter` resources. For me it makes no difference.

Reply:
Also as I know, we called always domains as apiGroups, not as mention for operated resources.